src/m_menu.c

```diff
@@ -512,6 +512,17 @@ static inline void M_DrawPatchInternal (int x, int y, int scrn, patch_t *patch)
     V_DrawPatchDirect (x, y, scrn, patch);
 }
 
+static inline void M_SetRelativeMode(boolean enabled) {
```

Comment:
Follow the original style. i.e.,
```c
static inline void M_SetRelativeMode(boolean enabled)
{
```

Reply:
I am not sure if the name `M_SetRelativeMode` might confuse someone who can not think of mouse operations.